indico/modules/users/export.py

```diff
@@ -298,6 +310,10 @@ def build_storage_path(file):
         event = file.registration.event
         prefix = 'registrations'
         path = f'{event.id}_{event.title}'
+    elif isinstance(file, ReceiptFile):
+        event = file.registration.event
+        prefix = 'documents'
+        path = f'{event.id}_{event.title}'
```

Comment:
Would it make sense to put this inside `registrations/.../documents/...` instead of it being another top-level folder?

Reply:
Yeah I agree, I changed such path